Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/ScreenData.purs

```diff
@@ -94,6 +94,14 @@ initData = {
       , index: 0
       , id: ""
       }
+    , tipViewState : {
+        primaryText : "Add a tip to find a ride quicker!"
+      , secondaryText : "It seems to be taking longer than usual."
+      , customerTipArray : ["₹10 🙂", "₹15 😄", "₹20 🤩"]
+      , customerTipArrayWithValues : [10, 15, 20]
+      , activeIndex : -1
+      , primaryButtonText : "Continue Search with +₹10 Tip"
+      }
```

Comment:
create a config in modal only with these default values, which you can overwrite as per need